packages/core/server/nakedRedirect.js

```diff
@@ -0,0 +1,28 @@
+const getConfigField = require('../utils/getConfigField');
+
+const nakedRedirect = (req, res, next) => {
+  const nakedRedirectConfig = getConfigField('nakedRedirect');
+
+  if (nakedRedirectConfig) {
+    let expressNakedRedirect;
+    try {
+      // Require the package.
+      expressNakedRedirect = require('express-naked-redirect'); // eslint-disable-line global-require
```

Comment:
Can you do the global require above, but just not call the function if the conditions aren't met? If that approach doesn't produce side effects, I'd recommend doing it that way.